tests/commands/test_runner.py

```diff
@@ -206,7 +206,7 @@ def test_sitecustomize_run_suppressed(self):
         # ensure `sitecustomize.py` is not loaded if `-S` is used
         env = inject_sitecustomize('tests/commands/bootstrap')
         out = subprocess.check_output(
-            ['ddtrace-run', 'python', 'tests/commands/ddtrace_run_sitecustomize.py', '-S'],
+            ['ddtrace-run', 'python', '-S', 'tests/commands/ddtrace_run_sitecustomize.py', '-S'],
```

Comment:
Should we remove the trailing `-S` argument?

Reply:
No, it's used in the script to make a different assertion.